frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -135,6 +135,10 @@ export const PublicFormProvider = ({
   const { isNotFormId, toast, vfnToastIdRef, expiryInMs, ...commonFormValues } =
     useCommonFormProvider(formId)
 
+  const isPaymentEnabled =
+    data?.form.responseMode === FormResponseMode.Encrypt &&
+    data.form.payments_field.enabled
```

Comment:
Ah dang it! This conflicts with the other PR that removes `payments_field.enabled`, and I still haven't had a chat with @staceytan1998 to decide on course of action for that 😰 . I suppose we could merge this first, and update the other PR later to make a change to this line if needed)

Reply:
Yep it does 😅, we will likely have another method to check the existence of payment.

I think Justyn and I both prefer checking the existence of `payments_field` so probably be like 
```
const isPaymentEnabled = 
    data?.form.responseMode === FormResponseMode.Encrypt &&
    data.form.payments_field
```

But will leave it to after discussion with hq and stacey!